Sources/AuthgearError.swift

```diff
@@ -24,12 +24,88 @@ public enum AuthgearError: Error {
     case unauthenticatedUser
     case publicKeyNotFound
     case error(Error)
+
+    public static var errorDomain: String { "AuthgearError" }
+    public var errorCode: Int {
+        switch self {
+        case .cancel:
+            return 0
+        case .unexpectedHttpStatusCode:
+            return 1
+        case .serverError:
+            return 2
+        case .oauthError:
+            return 3
+        case .anonymousUserNotFound:
+            return 4
+        case .biometricPrivateKeyNotFound:
+            return 5
+        case .biometricNotSupportedOrPermissionDenied:
+            return 6
+        case .biometricNoPasscode:
+            return 7
+        case .biometricNoEnrollment:
+            return 8
+        case .biometricLockout:
+            return 9
+        case .cannotReauthenticate:
+            return 10
+        case .invalidJWT:
+            return 11
+        case .unauthenticatedUser:
+            return 11
+        case .publicKeyNotFound:
+            return 11
+        case .error:
+            return 12
+        }
+    }
+
+    public var errorUserInfo: [String: Any] {
+        var info: [String: Any] = [:]
+        switch self {
+        case let .unexpectedHttpStatusCode(code, _):
+            info["code"] = code
+        case let .serverError(err):
+            info = err.errorUserInfo
+        case let .oauthError(err):
+            info = err.errorUserInfo
+        case let .biometricNotSupportedOrPermissionDenied(err):
+            info["error"] = err.localizedDescription
+        case let .biometricNoPasscode(err):
+            info["error"] = err.localizedDescription
+        case let .biometricNoEnrollment(err):
+            info["error"] = err.localizedDescription
+        case let .biometricLockout(err):
+            info["error"] = err.localizedDescription
+        case let .invalidJWT(jwt):
+            info["jwt"] = jwt
+        case let .error(err):
+            info["error"] = err.localizedDescription
+        default:
+            break
+        }
+        return info
+    }
 }
 
 public struct OAuthError: Error, Decodable {
```

Comment:
errorDecription: `${error}: ${errorDescription}`

Reply:
I found that we have property conflict with `LocalizedError` in OAuthError.
I think for OAuthError I will implement `localizedDescription` of `Error` instead.